js/util/is_char_in_unicode_block.js

```diff
@@ -0,0 +1,268 @@
+    'CJK Compatibility': (char) => char >= 0x3300 && char <= 0x33FF,
+    'CJK Unified Ideographs Extension A': (char) => char >= 0x3400 && char <= 0x4DBF,
+    // 'Yijing Hexagram Symbols': (char) => char >= 0x4DC0 && char <= 0x4DFF,
+    'CJK Unified Ideographs': (char) => char >= 0x4E00 && char <= 0x9FFF,
+    'Yi Syllables': (char) => char >= 0xA000 && char <= 0xA48F,
+    'Yi Radicals': (char) => char >= 0xA490 && char <= 0xA4CF,
+    // 'Lisu': (char) => char >= 0xA4D0 && char <= 0xA4FF,
+    // 'Vai': (char) => char >= 0xA500 && char <= 0xA63F,
+    // 'Cyrillic Extended-B': (char) => char >= 0xA640 && char <= 0xA69F,
+    // 'Bamum': (char) => char >= 0xA6A0 && char <= 0xA6FF,
+    // 'Modifier Tone Letters': (char) => char >= 0xA700 && char <= 0xA71F,
+    // 'Latin Extended-D': (char) => char >= 0xA720 && char <= 0xA7FF,
+    // 'Syloti Nagri': (char) => char >= 0xA800 && char <= 0xA82F,
+    // 'Common Indic Number Forms': (char) => char >= 0xA830 && char <= 0xA83F,
+    // 'Phags-pa': (char) => char >= 0xA840 && char <= 0xA87F,
+    // 'Saurashtra': (char) => char >= 0xA880 && char <= 0xA8DF,
+    // 'Devanagari Extended': (char) => char >= 0xA8E0 && char <= 0xA8FF,
+    // 'Kayah Li': (char) => char >= 0xA900 && char <= 0xA92F,
+    // 'Rejang': (char) => char >= 0xA930 && char <= 0xA95F,
+    // 'Hangul Jamo Extended-A': (char) => char >= 0xA960 && char <= 0xA97F,
+    // 'Javanese': (char) => char >= 0xA980 && char <= 0xA9DF,
+    // 'Myanmar Extended-B': (char) => char >= 0xA9E0 && char <= 0xA9FF,
+    // 'Cham': (char) => char >= 0xAA00 && char <= 0xAA5F,
+    // 'Myanmar Extended-A': (char) => char >= 0xAA60 && char <= 0xAA7F,
+    // 'Tai Viet': (char) => char >= 0xAA80 && char <= 0xAADF,
+    // 'Meetei Mayek Extensions': (char) => char >= 0xAAE0 && char <= 0xAAFF,
+    // 'Ethiopic Extended-A': (char) => char >= 0xAB00 && char <= 0xAB2F,
+    // 'Latin Extended-E': (char) => char >= 0xAB30 && char <= 0xAB6F,
+    // 'Cherokee Supplement': (char) => char >= 0xAB70 && char <= 0xABBF,
+    // 'Meetei Mayek': (char) => char >= 0xABC0 && char <= 0xABFF,
+    'Hangul Syllables': (char) => char >= 0xAC00 && char <= 0xD7AF,
+    // 'Hangul Jamo Extended-B': (char) => char >= 0xD7B0 && char <= 0xD7FF,
+    // 'High Surrogates': (char) => char >= 0xD800 && char <= 0xDB7F,
+    // 'High Private Use Surrogates': (char) => char >= 0xDB80 && char <= 0xDBFF,
+    // 'Low Surrogates': (char) => char >= 0xDC00 && char <= 0xDFFF,
+    // 'Private Use Area': (char) => char >= 0xE000 && char <= 0xF8FF,
+    'CJK Compatibility Ideographs': (char) => char >= 0xF900 && char <= 0xFAFF,
+    // 'Alphabetic Presentation Forms': (char) => char >= 0xFB00 && char <= 0xFB4F,
+    // 'Arabic Presentation Forms-A': (char) => char >= 0xFB50 && char <= 0xFDFF,
+    // 'Variation Selectors': (char) => char >= 0xFE00 && char <= 0xFE0F,
+    // 'Vertical Forms': (char) => char >= 0xFE10 && char <= 0xFE1F,
```

Comment:
Make this block vertical and ideographic. This block will also come in handy once we implement https://github.com/mapbox/mapbox-gl-js/pull/3438#issuecomment-255553240.